doc/src/manual/missing.md

```diff
@@ -296,6 +296,46 @@ since type constructors fall back to convert methods.
 Stacktrace:
 [...]
 ```
+## Skipping Missing Values
+
+Since `missing` values propagate with standard mathematical operators, reduction
+functions return `missing` when called on arrays which contain missing values:
+```doctest
+julia> sum([1, missing])
+missing
+
+```
+
+In this situation, use the [`skipmissing`](@ref) function to skip missing values:
+```doctest
+julia> sum(skipmissing([1, missing]))
+1
+
```

Comment:
Extra line?

Reply:
Also shouldn't these be ```jldoctest and not ```doctest?

Reply:
Thanks for spotting this! Actually, I had made the same mistake in the whole file...